### PR TITLE
Fixed `Event Methods` link in README.md

### PR DESCRIPTION
**Pull request by ArsenArsen**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ JDA jda = new JDABuilder(AccountType.BOT).setToken("token").buildBlocking();
 ## Events
 There a TON of events in JDA that you can listen to.<br>
 There are 2 ways of writing your Event-Listener:
-  1. Extend ListenerAdapter and use the provided methods that get fire dependent on the Event-Type. [Event Methods](https://github.com/DV8FromTheWorld/JDA/blob/master/src/main/java/net/dv8tion/jda/hooks/ListenerAdapter.java)
+  1. Extend ListenerAdapter and use the provided methods that get fire dependent on the Event-Type. [Event Methods](https://github.com/DV8FromTheWorld/JDA/blob/master/src/main/java/net/dv8tion/jda/core/hooks/ListenerAdapter.java)
```
Comment (DV8FromTheWorld) [inline README.md]: `fire` -> `fired`, `dependent` -> `depending`

**Comment by MinnDevelopment** [comment]
🤔 Should this base branch be `master`

**Comment by ArsenArsen** [comment]
It is @MinnDevelopment?

**Comment by MinnDevelopment** [comment]
Please reopen this PR to `development`. Then I will be able merge.

**Pull request closed without merging**